tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -191,10 +191,10 @@ protected GraphQLTests(string sampleAppName, ITestOutputHelper output, string te
         {
             SetServiceVersion(ServiceVersion);
             SetEnvironmentVariable("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", metadataSchemaVersion);
-#if NETCOREAPP2_1
-            // NET 2.1, sometimes times out
-            SetEnvironmentVariable(Configuration.ConfigurationKeys.ObfuscationQueryStringRegexTimeout, "600");
-#endif
+
+            // After updating the regex implementation multiple runtimes started hitting the a timeout, updating here to prevent flakes on master
+            SetEnvironmentVariable(Configuration.ConfigurationKeys.ObfuscationQueryStringRegexTimeout, "5_000");
```

Comment:
I wonder if we should set this for _all_ test, not just GraphQL. ASM tests in particular probably use querystrings, as do some aspnetcore test IIRC. But this is a good start!